src/main/java/racingcargame/domain/Cars.java

```diff
@@ -0,0 +1,35 @@
+package racingcargame.domain;
+
+import racingcargame.utils.RandomUtils;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    private int maxDistance() {
+        return cars.stream()
+                .mapToInt(Car::getDistance)
+                .max()
+                .orElseThrow(IllegalArgumentException::new);
+    }
+
+    public void race() {
+        cars.forEach(car -> car.move(RandomUtils.getRandomNumber()));
+    }
```

Comment:
```suggestion
    public void race(List<Integer> numbers) {
        for (int i = 0; i < numbers.size(); i++) {
            Car car = cars.get(i);
            car.move(numbers.get(i));
        }
    }
```

이러면 ```cars```와 ```randomNumber```와 의존성을 분리해 ```cars.race()```도 테스트할 수 있겠네요!